geomstats/geometry/minkowski.py

```diff
@@ -16,51 +17,40 @@ class Minkowski(Manifold):
     """
 
     def __init__(self, dim):
-        super(Minkowski, self).__init__(dim=dim)
+        super(Minkowski, self).__init__(dim=dim,
+                                        ambient_manifold=Euclidean(dim))
         self.metric = MinkowskiMetric(dim)
 
-    def belongs(self, point):
-        """Evaluate if a point belongs to the Minkowski space.
+    def belongs(self, point, atol=gs.atol):
+        """Evaluate if a point belongs to the manifold.
```

Comment:
to the Minkowski space? It would make the documentation website more specific when users browse to it.